src/scripts/articles/ArticlesProvider.js

```diff
@@ -0,0 +1,48 @@
+// Author: Calvin Curry
+// This module is responsible for providing the parsed article data from the database.JSON file. 
+
+
+const eventHub = document.querySelector("#container")
+
+
+const dispatchSavedArticle = () => {
```

Comment:
Too specific. Articles can also be deleted. This event should simply say that the state changed.